Submission of connectorio membership profile. (#104)
openhabfoundation/openhabfoundation.github.io: members/2024-01-07-connectorio.md

---
layout: MemberPage
category: cmember
logo : connectorio.png

title: 'ConnectorIO®'
address1: 'ConnectorIO sp. z o.o.'
address2: 'Nieszawska 4B'
city: '03-382 Warsaw'
country: 'Poland'
email: contact@connectorio.com
web: http://connectorio.com/oh
twitter: connectorio

---

> ConnectorIO® is service and technology provider focused on openHAB project. We completed number of deployments in Europe - Germany, Switzerland, Liechtenstein and Poland. Our firm is largest publisher of third party extensions for openHAB project. Addons which we support integrate variety of industrial and commercial real estate equipment.

Reviewer: ```suggestion
ConnectorIO® is a service and technology provider focused on openHAB project.
We have completed a number of deployments in Europe - Germany, Switzerland, Liechtenstein and Poland.
Our firm is the largest publisher of third party extensions for the openHAB project.
Add-ons which we support integrate a variety of industrial and commercial real estate equipment.
```

<!-- more -->

> ConnectorIO's focuses on safety of software and software integration towards production use.

Reviewer: ```suggestion
ConnectorIO focuses on safety of software and software integration towards production use.
```

> Over course of years we developed own [cloud connector](https://connectorio.com/services/cloud/connectorio-cloud/), which works independently of openHAB cloud as well as [energy monitoring solution](https://connectorio.com/solutions/energy-monitoring/) suited for resource constrained devices.

Reviewer: ```suggestion
Over the course of years we developed our own [cloud connector](https://connectorio.com/services/cloud/connectorio-cloud/), which works independently of openHAB cloud as well as an [energy monitoring solution](https://connectorio.com/solutions/energy-monitoring/) suited for resource constrained devices.
```

> We provide failsafe and hassle free upgrade mechanisms thanks to customized software assemblies and embedded linux distribution based on Yocto project.

Reviewer: ```suggestion
We provide a failsafe and hassle free upgrade mechanisms thanks to customized software assemblies and an embedded Linux distribution based on Yocto project.
```

> Our open source integrations work with BACnet, CAN bus, CANopen, OPC UA, Wireless M-Bus, M-Bus as well as PLC equipment manufactured by Beckhoff, Siemens and Fatek using their propertiary protocols.

Reviewer: ```suggestion
Our open source integrations work with BACnet, CAN bus, CANopen, OPC UA, Wireless M-Bus, M-Bus as well as with PLC equipment manufactured by Beckhoff, Siemens and Fatek using their proprietary protocols.
```

> **ConnectorIO is registered trademark of Code-House, Łukasz Dywicki**

Reviewer: ```suggestion
**ConnectorIO is a registered trademark of Code-House, Łukasz Dywicki**
```
